extensions/sql-database-projects/src/common/uiConstants.ts

```diff
@@ -9,6 +9,10 @@ export namespace cssStyles {
 	export const tableHeader = { ...text, 'text-align': 'left', 'border': 'none', 'font-size': '12px', 'font-weight': 'normal', 'color': '#666666' };
 	export const tableRow = { ...text, 'border-top': 'solid 1px #ccc', 'border-bottom': 'solid 1px #ccc', 'border-left': 'none', 'border-right': 'none', 'font-size': '12px' };
 	export const titleFontSize = 13;
+
 	export const publishDialogLabelWidth = '205px';
 	export const publishDialogTextboxWidth = '190px';
+
+	export const addDatabaseReferenceDialogLabelWidth = '215px';
+	export const addDatabaseReferenceInputboxWidth = '220px';
```

Comment:
Nit: should the inputbox and text box width be same?

Reply:
this is the label width and the textbox widths, so all the textboxes line up

Reply:
Sorry, I was asking about publishDialogTextboxWidth and addDatabaseReferenceInputboxWidth .

Reply:
the publish dialog text boxes have to be a little smaller because they have the icons next to them a couple of them, while the add database reference dialog doesn't have those and the text boxes can take up more space

Reply:
Makes sense. Thanks!